pkg/kubelet/network/dns/dns.go

```diff
@@ -283,8 +283,15 @@ func getPodDNSType(pod *v1.Pod) (podDNSType, error) {
 
 // Merge DNS options. If duplicated, entries given by PodDNSConfigOption will
 // overwrite the existing ones.
-func mergeDNSOptions(existingDNSConfigOptions []string, dnsConfigOptions []v1.PodDNSConfigOption) []string {
+func mergeDNSOptions(existingDNSConfigOptions []string, dnsConfigOptions []v1.PodDNSConfigOption, defaultDNSOptions []string) []string {
 	optionsMap := make(map[string]string)
+	for _, op := range defaultDNSOptions {
```

Comment:
This looks identical to the loop right below it. Why not just call `mergeDNSOptions(existingDNSConfig.Options, dnsConfig.Options)` and `mergeDNSOptions(dnsConfig.Options, defaultDNSOptions)`?

Reply:
Good point to move this to dnsConfig to get rid of code duplicate.